platform/README.md

```diff
@@ -0,0 +1,52 @@
+# New platform
+
+## Flow
+
+First install [`flow-typed`](https://github.com/flowtype/flow-typed):
+
+```
+npm install -g flow-typed
```

Comment:
Can we do this with a devDependency instead?  People shouldn't need to self-manage npm dependencies or use global installs.